docs/src/rsh/errors/index.md

```diff
@@ -1294,9 +1294,23 @@ You can fix this error by deleting one of the branches, ensuring there is only o
 This error indicates that a `{!rsh} switch` statement or `{!rsh} match` expression does not have a case
 for every variant of a `{!rsh} Data` instance.
 
+Example:
+
+```reach
+load: /hs/t/n/Err_Switch_MissingCases.rsh
+range: 11 - 13
+```
+
 You can fix this issue by adding the missing cases listed
 in the error message.
```

Comment:
It would be better to show a correct version of the exact same code... is that what that example is?